frontend/app_flowy/lib/plugins/grid/presentation/widgets/common/text_field.dart

```diff
@@ -18,6 +19,7 @@ class InputTextField extends StatefulWidget {
     this.onDone,
     required this.onCanceled,
     this.onChanged,
```

Comment:
I would add an optional FocusNode field here, and then in initState() down below, use the passed in FocusNode if it exists but create one if it doesn't. `_focusNode = widget.focusNode ?? FocusNode()`